### ISSUE TITLE
Revisit labeling required fields "required" 

### ISSUE THREAD

**Issue by Jacklynn** [18F/crt-portal#319]
[placeholder] we got a11y feedback that required fields should be labeled, the team should revisit this pattern considering what we learned considering the audience

- a11y: as a screen reader user, I should know which fields are required 
(In addition to other requirements already there: reading field label and showing active field, being able to tab to the field)

## Mocks
Moving towards indicating required fields only

![A Required - no optional.jpg](https://images.zenhubusercontent.com/5d1a3b0bcc794602e102a513/741fb3d7-1ee3-4bed-94b4-5b2aadc2d410)

### Typography
- **Text size** is always 1 rem, even if it's by larger text. 
- **asterisks:** 
color: darker (same as errors) `#b51d09` -  `red-60v`
font-family: PublicSans-BoldItalic; (bold)
- **required text:**
color: #757575 ‘gray-50’
font-family: PublicSans-Italic; (not bold)

### Spacing
If it's not too much work, I would expect .5rem for bigger titles (ex: blue text) and .3rem for regular field titles. If we'd prefer to keep it consistent, I would suggest the .3 rem for all of the spacing
![Screen Shot 2020-04-28 at 9.20.33 AM.png](https://images.zenhubusercontent.com/5d1a3b0bcc794602e102a513/8292d177-d870-4c0d-b1b4-5d2ea3171cf0)
![Screen Shot 2020-04-28 at 9.21.41 AM.png](https://images.zenhubusercontent.com/5d1a3b0bcc794602e102a513/a30f6195-c9c5-4d24-8cf6-8acd3d8fc9d0)

### Nice-to-have
On the date page, we were treating required/optional differently where it's just in the help text. Personally, I'd still like it visually indicated on this page as well. I would expect asterisks by the month & year and one by the help text as a description of what it means. 
![Screen Shot 2020-04-28 at 9.25.18 AM.png](https://images.zenhubusercontent.com/5d1a3b0bcc794602e102a513/9853866f-529b-4414-b796-ffdefe7acb46)

**Comment by cantsin** [comment]
It looks like we already display `(Optional)` text for _non_ required fields. I'm assuming we want to invert the logic here -- show `* required` and remove `(Optional)`?

**Comment by Jacklynn** [comment]
Correct!

**Comment by blaughman** [comment]
Function is as intended.  Awaiting confirmation from @Jacklynn before moving to UAT.

**Comment by Jacklynn** [comment]
@cantsin On the Location details page, can we remove the required label after the "where did this happen?" blue text? Only some of the fields in this group are required. 

With a screen reader, it reads the group name afterwards which includes the "required" text so it makes it seem like all the labels are required. 

This is the page after the hate crimes & human trafficking page. This section shows up for any primary issue that is selected.
![Screen Shot 2020-05-11 at 8.38.04 AM.png](https://images.zenhubusercontent.com/5d1a3b0bcc794602e102a513/dd2c9a58-646b-4b8b-be12-9a7bbafd22b7)

**Comment by cantsin** [comment]
Sure thing! In that case I'll make a comment saying why in the code -- don't want to accidentally add this back down the road.

**Comment by blaughman** [comment]
PO approves.  @Jacklynn I assume this needs to be checked with Terri Youngblood before we move it along?

**Comment by Jacklynn** [comment]
Terri would have gone over this as part of her review on Thursday and she didn't mention anything!

**Comment by Jacklynn** [comment]
Everything looked good to me
